scripts/override_images/override_os_images.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Overrides a given list of OS images to a list with a single OS image
+that matches the latest release image in RELEASE_IMAGES
+"""
+import json
+import os
+import sys
+from argparse import ArgumentParser
+
+from assisted_service_client.models import OsImage, ReleaseImage
+
+from assisted_test_infra.test_infra import utils
```

Comment:
we are only using on function
```suggestion
from assisted_test_infra.test_infra.utils import extract_version
```

Reply:
semantic change :)